### ISSUE TITLE
inline essentially causes the code to load twice?

### ISSUE THREAD

**Issue by whats1thingnow** [bezoerb/grunt-critical#22]
as per readme:

> If dest points to an Markup file (HTML, PHP, etc.) the resulting CSS gets inlined and the exiting stylesheets are wrapped in a javascript function to load them asynchronously as well as a noscript block for users with javascript disabled

so that means, the critical pieces will be loaded the 1st time inside of the html

then, 2nd, the same critical pieces will be loaded again asynchronously (along with the non critical)?

thank you for clarifying!

**Comment by bezoerb** [comment]
@whats1thingnow that's true. Critical also supports an `extract`option which removes the critical css from the source stylesheets. Keep in mind that this option might create different source stylesheets for different pages which will drop any caching benefits. See https://github.com/addyosmani/critical/issues/39 for further insights
